process_multiple_plates.py

```diff
@@ -0,0 +1,53 @@
+import os
+import pathlib
+import pickle
+import cv2
+
+import imagej
+import skimage
+import numpy as np
+from IPython.utils.io import capture_output
+
+import warnings
+import PyBaSiC.pybasic as pybasic
+
+from preprocessor import BasicpyPreprocessor
+
+
+
+fiji_path = pathlib.Path("/home/keenanmanpearl/Desktop/mitocheck/fiji-linux64/Fiji.app")
+movie_dir = pathlib.Path("/home/keenanmanpearl/Desktop/mitocheck_movies/movies/")
+fiji = BasicpyPreprocessor(fiji_path)
+frames = 10
+
+for plate_dir in movie_dir.iterdir():
+    for well_dir in plate_dir.iterdir():
+        for movie_path in well_dir.iterdir():
+            print(movie_path)
+            save_path = movie_path.with_suffix(".avi")
+            movie = fiji.load_mitocheck_movie_data(movie_path)
+            corrected_movie = fiji.pybasic_illumination_correction(movie)
+            frames = 10
+            #frames = len(corrected_movie)
+            # height = 1024
+            height = len(corrected_movie[0])
+            # width = 1344
+            width = len(corrected_movie[0][0])
+            down_height = int(height/20)
+            down_width = int(width/20)
```

Comment:
Can you add a comment about what these are doing?

Also, you probably should make `20` a specifially named variable somewhere. (It will help with code understandability too)